packages/legend-extension-dsl-persistence/src/graph/metamodel/pure/model/packageableElements/persistence/DSL_Persistence_Persistence.ts

```diff
@@ -45,7 +47,8 @@ export class Persistence
       this.documentation,
       this.trigger,
       this.service.valueForSerialization ?? '',
-      this.persister,
+      hashArray(this.serviceOutputTargets ?? []),
```

Comment:
Please make the similar modification as mentioned in protocol model

Reply:
As discussed keeping it as empty array [].